grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -2814,4 +2814,18 @@ class StockMovementService {
         }
         return true
     }
+
+    void deleteComment(Comment comment, Requisition requisition) {
+        requisition.removeFromComments(comment)
+    }
+
+    void saveComment(Comment comment) {
+        comment.save(failOnError: true)
+    }
+
+    void addCommentToRequisition(Comment comment, Requisition requisition) {
+        comment.save(failOnError: true)
```

Comment:
I think this is unnecessary - the `requisition.save()` should handle this by cascade - maybe instead of `failOnError` let's do a `validate()` and then eventually return `comment.errors` to the view instead of throwing a `ValidationException`?